README.md

```diff
@@ -1,178 +1,229 @@
+      console.log(suite.table.toString());
+    })
+    // Run async
+    .run({ async: false })
+    ;
+
+// =>
+// RegExp#test for inputs Short string x 11,037,873 ops/sec ±0.57% (100 runs sampled)
+// RegExp#test for inputs Long string x 10,114,587 ops/sec ±0.42% (100 runs sampled)
+// RegExp#test for inputs Very long string x 7,534,743 ops/sec ±0.33% (101 runs sampled)
+// RegExp#test for inputs Extremely long string x 304,666 ops/sec ±0.25% (101 runs sampled)
+// String#indexOf for inputs Short string x 13,400,084 ops/sec ±0.28% (99 runs sampled)
+// String#indexOf for inputs Long string x 12,947,759 ops/sec ±0.33% (100 runs sampled)
+// String#indexOf for inputs Very long string x 8,489,440 ops/sec ±0.31% (101 runs sampled)
+// String#indexOf for inputs Extremely long string x 306,018 ops/sec ±0.26% (100 runs sampled)
+// Fastest is String#indexOf for inputs Short string
+// +----------------+--------------------+--------------------+-------------------+-----------------------+
+// |                │ Short string       │ Long string        │ Very long string  │ Extremely long string |
+// +----------------+--------------------+--------------------+-------------------+-----------------------+
+// | RegExp#test    │ 11,037,873 ops/sec │ 10,114,587 ops/sec │ 7,534,743 ops/sec │ 304,666 ops/sec       |
+// +----------------+--------------------+--------------------+-------------------+-----------------------+
+// | String#indexOf │ 13,400,084 ops/sec │ 12,947,759 ops/sec │ 8,489,440 ops/sec │ 306,018 ops/sec       |
+// +----------------+--------------------+--------------------+-------------------+-----------------------+
+```
+
+## Transposed table
+
+If the Benchtable object is initialized with the `{isTransposed : true}` option (see [this example](/examples/string_search_transposed.js)), the script produces the same output but with rows and columns transposed:
 
-```javascript
+```js
 // import module
-var Benchtable = require('benchtable');
-
-// create benchtable suite
-var suite = new Benchtable();
-
-// add functions
-suite.addFunction('RegExp#test', function(s) { /o/.test(s); })
-.addFunction('String#indexOf', function(s) {s.indexOf('o') > -1;})
-
-// add inputs
-.addInput('Short string', ['Hello world!'])
-.addInput('Long string', ['This is a very big string, isnt it? It is. Really. So, hello world!'])
-.addInput('Very long string', ['This is a ' + new Array(200).join("very ") + 'big string, isnt it? It is. Really. So, hello world!'])
-.addInput('Extremely long string', ['This is a ' + new Array(20000).join("very ") + 'big string, isnt it? It is. Really. So, hello world!'])
-
-// add listeners
-.on('cycle', function(event) {
-  console.log(String(event.target));
-})
-.on('complete', function() {
-  console.log('Fastest is ' + this.filter('fastest').pluck('name'));
-  console.log(this.table.toString());
-})
-
-// run async
-.run({ 'async': false });
+// Import module
+var BenchTable = require('benchtable'); //require('benchtable');
```

Comment:
I didn't. `blah` added it. :smile: I removed it completely from example files in 83d282d491bc886487ecb376f36b519aa9c408f7.